scipy/stats/_continuous_distns.py

```diff
@@ -3088,7 +3114,11 @@ def _fitstart(self, data):
         # of the data.  The formula is regularized with 1e-8 in the
         # denominator to allow for degenerate data where the skewness
         # is close to 0.
-        a = 4 / (1e-8 + _skew(data)**2)
+        if isinstance(data, CensoredData):
+            sk = _skew(data._uncensor())
```

Comment:
Why not just prepend:
```python3
        if isinstance(data, CensoredData):
            data = data._uncensor()
```
like everywhere else?
`super()._fitstart` is going to uncensor it again, anyway, right?

Reply:
Yup, fixed in https://github.com/scipy/scipy/pull/13699/commits/e93659aac9090da173644afc8a27fb9137525d38